### PR TITLE
fix quantize_squash_pass segfault when no tensor linked to Bias

### PR DESCRIPTION
**Pull request by sfraczek**
A fix for googlenet when there is zero vector for Bias input and we try to take the first element from it.

### CODE REVIEW

**Comment by luotao1** [comment]
@sfraczek Could you add googlenet UT here?

File: paddle/fluid/inference/tests/api/CMakeLists.txt [hunk 1]
```diff
@@ -194,6 +194,13 @@ if(WITH_MKLDNN)
     inference_download_and_uncompress(${INT8_VGG19_MODEL_DIR} "${INFERENCE_URL}/int8" "VGG19_int8_model.tar.gz" )
   endif()
   inference_analysis_api_int8_test(test_analyzer_int8_vgg19 ${INT8_VGG19_MODEL_DIR} ${INT8_DATA_DIR} analyzer_int8_image_classification_tester.cc)
+
+  #googlenet int8
+  set(INT8_GOOGLENET_MODEL_DIR "${INT8_DATA_DIR}/googlenet")
+  if (NOT EXISTS ${INT8_GOOGLENET_MODEL_DIR})
+    inference_download_and_uncompress(${INT8_GOOGLENET_MODEL_DIR} "${INFERENCE_URL}/int8" "GoogleNet_int8_model.tar.gz" )
+  endif()
+  inference_analysis_api_int8_test(test_analyzer_int8_googlenet ${INT8_GOOGLENET_MODEL_DIR} ${INT8_DATA_DIR} analyzer_int8_image_classification_tester.cc SERIAL)
```
Comment (lidanqing-vv) [inline paddle/fluid/inference/tests/api/CMakeLists.txt:203]: Hi, @sfraczek  remove this SERIAL. I forgot to delete it when I sent you.

Comment (sfraczek) [reply paddle/fluid/inference/tests/api/CMakeLists.txt:203]: ok

**Comment by luotao1** [comment]
```
test_analyzer_int8_googlenet ..............***Exception
[01:19:14]F0523 01:19:12.064352 11918 tester_helper.h:467] Check failed: std::abs(avg_acc1_quant - avg_acc1_ref) <= FLAGS_quantized_accuracy (0.72 vs. 0.01) 
[01:19:14]*** Check failure stack trace: ***
```

**Comment by sfraczek** [comment]
We are investigating why it doesn't work on this test. It works fine on our C-API application with the same configuration.

**Comment by sfraczek** [comment]
I fixed it. Waiting for CI checks.

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**